clippy_lints/src/copy_refcell.rs

```diff
@@ -0,0 +1,112 @@
+use clippy_config::Conf;
+use rustc_hir::{FieldDef, LetStmt, LocalSource};
+use rustc_lint::{LateContext, LateLintPass};
+use rustc_middle::ty;
+use rustc_middle::ty::layout::TyAndLayout;
+use rustc_session::impl_lint_pass;
+use rustc_span::symbol::sym;
+use rustc_span::Span;
+
+use clippy_utils::diagnostics::span_lint_and_help;
+use clippy_utils::ty::implements_trait;
+
+declare_clippy_lint! {
+    /// ### What it does
+    ///
+    /// Detects crate-local usage of `RefCell<T>` where `T` implements `Copy`
```

Comment:
imo saying it's crate-local is extraneous. I also think that the whole "is local" part of the original issue is regarding the size issue, and it being slower. This is already checked so I believe it is fine to forego that?

Reply:
If not, the check for it being local should be added :P

Reply:
Ah yes, I completely forgot to actually implement crate-local! I've added the publicity check for fields now.